src/System.Runtime/tests/Performance/Perf.String.netcoreapp.cs

```diff
@@ -10,15 +10,18 @@ public partial class Perf_String
 
         [Benchmark]
         [MemberData(nameof(ContainsStringComparisonArgs))]
-        public void Contains(StringComparison comparisonType, int size)
+        public void Contains(StringComparison comparisonType, int[] sizes)
```

Comment:
I think the test should expect that only a single size is given to it. The MemberData method (ContainsStringComparisonArgs) should probably return individual test values rather than arrays.